### PR TITLE
Feat/add datahub for plain implementation

### PR DESCRIPTION
**Pull request by BernardTolosajr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +62,7 @@
     "less": "^2.7.3",
     "less-loader": "^4.0.5",
     "macaca-cli": "^2.0.12",
+    "macaca-datahub": "^1.0.74",
```
Comment (xudafeng) [inline package.json]: please use datahub2

"macaca-datahub": "^1.0.74",     =>  "macaca-datahub": "2",

**Comment by xudafeng** [comment]
@BernardTolosajr 

please use DataHubv2

### How to migrate data from v1 to v2？

```
./node_modules/.bin/datahub-migrate-1-to-2 -s ./data 
```

**Comment by BernardTolosajr** [comment]
@xudafeng done migrating to datahub 2

File: package.json [hunk 2]
```diff
@@ -75,6 +77,7 @@
     "react-redux": "^5.0.6",
     "redux": "^3.7.2",
     "redux-devtools": "^3.4.1",
+    "sqlite3": "^4.0.2",
```
Comment (xudafeng) [inline package.json]: not need

**Pull request merged**